features/journey_hint.feature

```diff
@@ -21,33 +21,3 @@ Scenario: Journey hint Registration
     And they select journey hint "Sign-in with eIDAS"
     And they start a journey
     Then they should arrive at the country picker page
-
-  Scenario: Journey hint Sign-in with eIDAS when Eidas is Disabled
```

Comment:
all of this are unit tests pretty much covered in https://github.com/alphagov/verify-frontend/blob/master/spec/features/redirects_with_journey_hint_authn_request_spec.rb